tdrs-frontend/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     ports:
       - 8090:8090
     networks:
-      - local
+      - default
```

Comment:
I'd like to invite others to comment on this change. I removed the local network because I found it obstructing container comm in my local environment. It must have been put here for a reason, though. Can someone provide insight on the local and external-net networks and how they were meant to be used by frontend and backend containers?